tools/querytee/response_comparator_test.go

```diff
@@ -2179,6 +2179,18 @@ func TestCompareSamplesResponse(t *testing.T) {
 						}`),
 			skipSamplesBefore: 95 * 1000,
 		},
+		{
+			name: "should not fail when some series is entirely skipped and rest matches - float",
```

Comment:
Could you please also add a test for the case where a series is entirely skipped in one response, but some samples are not skipped in the other response?

eg. if `skipSamplesBefore` is 95, then series A has samples at 90 and 93 in the expected response, and samples at 90, 93 and 100 in the actual response

Reply:
Done! I also added warning to the error message like we are doing for vector responses.